packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md

```diff
@@ -0,0 +1,299 @@
+# Mapping Table Column
+
+## Overview
+
+The `nimble-table-column-mapping` is a component that supports rendering specific number, boolean, or string values as mapped icons, spinners, or text. `nimble-table-column-icon` is a specialized version of `nimble-table-column-mapping` which defaults to having a minimal, fixed width. The actual mappings are defined by child elements `nimble-mapping-icon`, `nimble-mapping-spinner`, and `nimble-mapping-text`.
```

Comment:
> The `nimble-table-column-mapping` is a component that supports rendering specific number, boolean, or string values as mapped icons, spinners, or text.

Technically the `nimble-table-column-mapping` isn't what supports a value being mapped to a icon/spinner, right? It maps to text, which may have an optional icon, right? I think it would be good to tighten up the language here a little bit.

Oh, I actually see below that you've called out _icon + text_ as out of scope. In that case, then I definitely think this wording should be updated since the mapping column doesn't support icons or spinners at all.

Reply:
I guess I had been thinking the `nimble-table-column-icon` was a convenience element, but hadn't made the jump to limiting the general mapping column to just text. That would make sense though, and would avoid ambiguity about which column type to use.